src/main/java/wordle/model/Words.java

```diff
@@ -0,0 +1,26 @@
+package wordle.model;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class Words {
+
+    private final List<String> words;
+    private final LocalDate cutoffDate;
+
+    public Words(List<String> wordList, LocalDate cutoffDate) {
+        this.words = wordList;
+        this.cutoffDate = cutoffDate;
+    }
+
+    public String getWordOfDay(LocalDate localDate) {
+        long nowEpochDay = localDate.toEpochDay();
+        long cutoffEpochDay = cutoffDate.toEpochDay();
+        int wordIndex = (int) (nowEpochDay - cutoffEpochDay) % words.size();
+        return words.get(wordIndex);
```

Comment:
```suggestion
        int diffDay = Period.between( (int) cutoDate.toEpocDay(), localDate.toEpochDay()).getDays();
        int index = diffDay % words,size();
```
와 같이 Period.between을 사용해`(nowEpochDay - cutoffEpochDay)` 부분을 대체할 수도 있을 거 같네요!

Reply:
오 모르고 있던 클래스인데 훨씬 깔끔해지네요! 알려주셔서 감사합니다 👍🏻 👍🏻 👍🏻 
바로 적용하러 갑니닷

Reply:
오 모르고 있던 클래스인데 훨씬 깔끔해지네요! 알려주셔서 감사합니다 👍🏻 👍🏻 👍🏻
바로 적용했습니다.